src/main/java/org/broadinstitute/hellbender/Main.java

```diff
@@ -117,24 +138,19 @@ protected final void mainEntry(final String[] args) {
             handleResult(result);
             System.exit(0);
         } catch (final CommandLineException e){
-            //the usage has already been printed so don't print it here.
-            if(printStackTraceOnUserExceptions()) {
-                e.printStackTrace();
-            }
+            handleUserException(e);
```

Comment:
Right - Barclay will be changed soon so that CommandLineException can always be treated as a user error, and internal exception will be separate. Once thats done I think this will do exactly the right thing.